cmd/ltctl/main.go

```diff
@@ -47,6 +47,16 @@ func RunDestroyCmdF(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create terraform engine: %w", err)
 	}
+
+	status, err := t.DBStatus()
+	if err != nil {
+		return fmt.Errorf("failed to get DB status: %w", err)
+	}
+
+	if status != dbAvailable {
+		return fmt.Errorf("The database isn't available at the moment. Its status is %q. Please check this, and then try again", status)
+	}
+
```

Comment:
We should probably run this only if there is a deployed DB. If not, we're using an external one and the destroy will not affect it in any case.